intel-sgx/pcs/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "pcs"
+version = "0.1.0"
+authors = ["Raoul Strackx <raoul.strackx@fortanix.com>"]
```

Comment:
Nit: worth adding @nshyrei to one of authors (authors field also could contain maintainers)